### ISSUE TITLE
can't support relative path after 2.30

### ISSUE THREAD

**Issue by fcying** [trapexit/mergerfs#879]
**Describe the bug**
in 2.29, I use mergerfs for some relative path. ex:
```
mkdir a b c
touch a/a b/b
mergerfs ./a:./b ./c
```
It work fine.
After I update to 2.32, it can't work, only work for Absolute path.
```
mergerfs ./a:./b ./c
* ERROR: unable to parse 'branches' - ./a:./b
* ERROR: unable to parse 'branches' - ./c
* ERROR: branches not set
* ERROR: mountpoint not set

mergerfs /mnt/a:/mnt/b /mnt/c
```
Was it designed this way on purpose? Or something went wrong.

**System information:**
 - OS, kernel version: `Linux ve 5.4.78-2-pve #1 SMP PVE 5.4.78-2 (Thu, 03 Dec 2020 14:26:17 +0100) x86_64 GNU/Linux`
 - mergerfs version: `2.32.2`

**Comment by trapexit** [comment]
I did refactor things but I guess I broke something in the parsing. Sorry about that. Will take a look.

**Comment by trapexit** [comment]
Fixed by #880. New version released.

**Comment by fcying** [comment]
It can work now, thanks~
